pybuildkite/buildkite.py

```diff
@@ -4,7 +4,11 @@
 from pybuildkite.builds import Builds, BuildState
 from pybuildkite.jobs import Jobs
 from pybuildkite.agents import Agents
+<<<<<<< HEAD
 from pybuildkite.emojis import Emojis
+=======
+from pybuildkite.annotations import Annotations
+>>>>>>> ✨ Added. Annotation Class
```

Comment:
Merge conflicts :)

Reply:
I didn't see anythin. Could you take a look again? xD 

I rebase my branch with commit on master and all seems to be ok.

Reply:
The characters in this file indicate a merge conflict that needs to be resolved before we can merge. Check out the <<<<<<head

Reply:
Done ^^ sorry for my blindness